packages/turf-isolines/index.ts

```diff
@@ -1,8 +1,9 @@
 import bbox from "@turf/bbox";
+import clone from "@turf/clone";
 import { coordEach } from "@turf/meta";
 import { collectionOf } from "@turf/invariant";
 import { multiLineString, featureCollection, isObject } from "@turf/helpers";
-import isoContours from "./lib/marchingsquares-isocontours";
```

Comment:
you probably also want to remove the `./lib/marchingsquares-isocontours.js` file